create_builder_test.go

```diff
@@ -485,6 +485,21 @@ func testCreateBuilder(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
+		when("package file", func() {
+			it("package file is valid", func() {
+				cnbFile := filepath.Join("testdata", "bp_one.cnb")
+
+				mockDownloader.EXPECT().Download(gomock.Any(), cnbFile).Return(blob.NewBlob(cnbFile), nil).AnyTimes()
```

Comment:
To the point above re: fragility, is there a way to assert that `AddBuildpack` was called on the file?

Reply:
Add verification that it's on the final builder!